geomstats/geometry/pullback_metric.py

```diff
@@ -477,6 +477,43 @@ def dist(self, point_a, point_b, **kwargs):
         image_point_b = self.diffeomorphism(point_b)
         return self.embedding_space.metric.dist(image_point_a, image_point_b, **kwargs)
 
+    def geodesic(self, initial_point, end_point=None, initial_tangent_vec=None):
+        """Compute the geodesic via diffeomorphic pullback.
+
+        Parameters
+        ----------
+        point_a : array-like, shape=[..., *shape]
+            Point a.
+        point_b : array-like, shape=[..., *shape]
+            Point b.
+
+        Returns
+        -------
+        geodesic : callable
+            Geodesic between point_a and point_b.
```

Comment:
Idem: it's either between initial point and end point, or with initial point and initial tangent vec.